Airports/EGGD/SMR/Regions.txt

```diff
@@ -1229,3 +1176,38 @@ smrGDBuilding N051.22.49.680 W002.43.21.911
 N051.22.50.836 W002.43.22.076
 N051.22.50.758 W002.43.24.330
 N051.22.49.593 W002.43.24.246
+
+; - Grey_Background (New stand 18)
+REGIONNAME Bristol
+smrGDapron N051.23.07.355 W002.42.08.068
+N051.23.07.789 W002.42.09.342
+N051.23.08.284 W002.42.09.722
+N051.23.09.119 W002.42.11.318
+N051.23.08.930 W002.42.19.306
+N051.23.06.564 W002.42.19.092
+N051.23.06.775 W002.42.07.993
+
+; - Bottom_conc
+REGIONNAME Bristol
+smrGDapron N051.23.06.401 W002.42.19.032
+N051.23.05.958 W002.42.18.970
+N051.23.06.204 W002.42.07.813
+N051.23.06.867 W002.42.08.015
+N051.23.06.556 W002.42.19.054
+
+; - Bottom_conc_sec
+REGIONNAME Bristol
+smrGDapron N051.23.05.877 W002.42.18.786
+N051.23.05.759 W002.42.18.556
+N051.23.05.645 W002.42.18.324
+N051.23.05.619 W002.42.18.004
+N051.23.05.640 W002.42.17.020
+N051.23.05.661 W002.42.16.512
+N051.23.05.758 W002.42.16.212
+N051.23.05.851 W002.42.15.909
+N051.23.06.061 W002.42.15.451
+N051.23.05.951 W002.42.18.952
+
+
+
+
```

Comment:
Same here regarding extra blank lines. Also for next time you could have probably done one 4-corner region and popped it in the right place in the draw order to have the same effect and save some lines, but it looks fine on the SMR so not to worry :)